mjpc/tasks/humanoid/interact/contact_keyframe.h

```diff
@@ -41,22 +49,38 @@ class ContactPair {
         local_pos2{0.} {}
 
   void Reset();
+  void GetDistance(mjtNum distance[3], const mjData* data) const;
 };
 
 class ContactKeyframe {
  public:
   std::string name;
   ContactPair contact_pairs[kNumberOfContactPairsInteract];
-
   // the direction on the xy-plane for the torso to point towards
   std::vector<mjtNum> facing_target;
 
   // weight of all residual terms (name -> value map)
   std::map<std::string, mjtNum> weight;
 
-  ContactKeyframe() : name(""), contact_pairs{}, facing_target(), weight() {}
+  ContactKeyframe()
```

Comment:
Is this the correct formatting?

Reply:
Yes, it matches the clang-format google style.